client/my-sites/site-settings/form-writing.jsx

```diff
@@ -104,7 +104,7 @@ class SiteSettingsFormWriting extends Component {
 
 							<CustomContentTypes
 								onSubmitForm={ this.props.handleSubmitForm }
-								handleToggle={ handleToggle }
+								handleToggle={ handleAutosavingToggle }
```

Comment:
I think this would only happen if you still remember how things are done internally with wrap-settings-form. If you start by looking at the `<CustomContentTypes` component, `handleToggle` does not say anything in particular about it being autosaved or not, or even where it's coming from.. But I'll make the change you propose